lxd/instance/drivers/driver_qemu.go

```diff
@@ -2980,6 +2980,29 @@ func (d *qemu) addRootDriveConfig(mountInfo *storagePools.MountInfo, bootIndexes
 		TargetPath: rootDriveConf.TargetPath,
 	}
 
+	if d.storagePool.Driver().Info().Remote {
+		vol := d.storagePool.GetVolume(storageDrivers.VolumeTypeVM, storageDrivers.ContentTypeBlock, project.Instance(d.project, d.name), nil)
+
+		config := d.storagePool.ToAPI().Config
+
```

Comment:
Do we need to do the `optEscaper := strings.NewReplacer(":", `\:`, "@", `\@`, "=", `\=`)` logic here too (once its in its own function)?